contracts/contracts/GrantRegistry.sol

```diff
@@ -0,0 +1,135 @@
+pragma solidity ^0.8.5;
+
+contract GrantRegistry {
+  // Grant ID - The hash of the grants meetadata
+  bytes32 public grantRegistryId;
+  // Metadata Pointer
+  string public metaPtr;
+
+  // Owner
+  address owner;
```

Comment:
I agree we probably do want that. For something as simple as ownership, we might want to just implement it ourselves, even if we copy/paste some of the `Ownable` code. Just to cut down on dependencies and contract junk/bloat (like `Context` and unnecessary getters)

------------------------

Also we probably want to make `owner` public. When not specified, it defaults to private

Reply:
I added this when I was considering deploying a contract for each grant. I have since changed the approach to having a single contract which manages many grants.

We can remove this.